pilot/pkg/model/context.go

```diff
@@ -356,8 +359,10 @@ type WatchedResource struct {
 	NonceNacked string
 
 	// AlwaysRespond, if true, will ensure that even when a request would otherwise be treated as an
-	// ACK, it will be responded to. Typically, this should be set to 'false' after response; keeping it
-	// true would likely result in an endless loop.
+	// ACK, it will be responded to. This typically happens when a proxy reconnects to another instance of
+	// Istiod. In that case, Envoy expects us to respond to EDS/RDS/SDS requests to finish warming of
```

Comment:
This exclusive applies to CDS+EDS as far as I know, not all dependent types

Reply:
> Can you give an example Istiod ads:debug log of this happening?

I have n't seen this happen in Istiod. But based on the above it should apply to any named resource. Let me try to setup a test case for it.